src/common/utils/crypto.utils.ts

```diff
@@ -0,0 +1,29 @@
+import { randomBytes, scrypt, timingSafeEqual } from 'crypto'
```

Comment:
Let's use modern prefixed syntax when importing native Node modules:
```suggestion
import { randomBytes, scrypt, timingSafeEqual } from 'node:crypto'
```

Reply:
Fixed